contracts/interfaces/IUniswapV3Staker.sol

```diff
@@ -11,30 +17,49 @@ interface IUniswapV3Staker {
         uint32 claimDeadline,
         uint128 indexed totalReward
     );
+
+    /// @notice Event emitted when a liquidity mining incentive has ended
+    /// @param rewardToken The address of the token being distributed as a reward
+    /// @param pool The address of the Uniswap V3 pool
+    /// @param startTime The time when the incentive program begins
+    /// @param endTime The time when rewards stop accruing
     event IncentiveEnded(
         address indexed rewardToken,
         address indexed pool,
         uint32 startTime,
         uint32 endTime
     );
 
+    /// @notice Event emitted when a Uniswap V3 LP token has been deposited
+    /// @param tokenId The unique identifier of an Uniswap V3 LP token
+    /// @param owner The owner of the LP token
     event TokenDeposited(uint256 tokenId, address indexed owner);
+
+    /// @notice Event emitted when a Uniswap V3 LP token has been withdrawn
+    /// @param tokenId The unique identifier of an Uniswap V3 LP token
+    /// @param to The address that the token will be withdawn to
     event TokenWithdrawn(uint256 tokenId, address to);
 
+    /// @notice Event emitted when a Uniswap V3 LP token has been staked
+    /// @param tokenId The unique identifier of an Uniswap V3 LP token
+    /// @param liquidity The amount of liquidity staked
     event TokenStaked(uint256 tokenId, uint128 liquidity);
-    event TokenUnstaked(uint256 tokenId);
 
-    event RewardClaimed(address indexed user, uint256 reward);
-
-    /**
-    @param rewardToken The token being distributed as a reward
-    @param pool The Uniswap V3 pool
-    @param startTime When rewards should begin accruing
-    @param endTime When rewards stop accruing
-    @param claimDeadline When program should expire
-    @param totalReward Total reward to be distributed
-    */
+    /// @notice Event emitted when a Uniswap V3 LP token has been unstaked
+    /// @param tokenId The unique identifier of an Uniswap V3 LP token
+    event TokenUnstaked(uint256 tokenId);
 
+    /// @notice Event emitted when a reward token has been claimed
+    /// @param to The address where claimed rewards were sent to
+    /// @param reward The amount of reward tokens claimed
+    event RewardClaimed(address indexed to, uint256 reward);
+
+    /// @param rewardToken The address of the token being distributed as a reward
+    /// @param pool The address of the Uniswap V3 pool
+    /// @param startTime The time when the incentive program begins
+    /// @param endTime The time when rewards stop accruing
+    /// @param claimDeadline -
```

Comment:
Wordy but I'd describe it as:

Time after which LPs can no longer claim rewards (and  `incentiveCreator` can end the incentive and receive unclaimed rewards)